### ISSUE TITLE
Issue with Vamp-IR tuples

### ISSUE THREAD

**Issue by CarloModicaPortfolio** [anoma/vamp-ir#88]
"(" and ")" are not in fact needed to write tuples with Vamp-IR. For instance:
```
def a = 1,2,3,4,5;
def b = (1,2,3,4,5);
a = b;
```
gives a valid proof

**Comment by AHartNtkn** [comment]
I don't think this is a real issue. As it stands, `,` is a left-associative binary operation that produces a tuple. Unless that causes problems, I don't see any reason why parens would have to be enforced.
